Translations/Web/lang_bg.php

```diff
@@ -22,7 +22,7 @@
 $content_title = "macOS файлов архиватор";
 
 // Download
-$content_plataform = "Изисква $version_replace или по-нова";
```

Comment:
I'm ashamed 🤦🏻‍♂️

Reply:
From the Spanish "plataforma"...

Reply:
> From the Spanish "plataforma"...

I immediately realized that you took it from your native language :)